src/generators/legacy-json/constants.mjs

```diff
@@ -0,0 +1,30 @@
+/**
+ * Denotes a method's return value
+ */
```

Comment:
<details>
<summary>Nitpick alert 😄 </summary>
For example, in the file <code>node-api-docs-tooling/src/constants.mjs</code>, comments are added in the format below;

```mjs
// Some comment about the constant
export const CONSTANT_NAME = 'constant_value';
```
It might be better to proceed in a standard way throughout the project
</details>